src/sentry/static/sentry/app/views/settings/components/settingsBackButton.jsx

```diff
@@ -29,7 +29,7 @@ const Icon = styled(InlineSvg)`
 
 class BackButton extends React.Component {
   static propTypes = {
-    organization: SentryTypes.Organization,
+    organization: PropTypes.object,
```

Comment:
Does this need to be changed?

Reply:
The organization.id field is required, which creates a false warning now that users can get to account settings without an organization. What do you think about removing `isRequired` here: https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/sentryTypes.jsx#L308

Reply:
Wouldn't organization be undefined in that case?

Reply:
Hm... ya I think you're right. Not sure why I was seeing that warning before. Will change it back.